Add callouts about App Store restrictions (#861)
vtexdocs/dev-portal-content: docs/release-notes/extensions-hub-our-new-place-to-promote-apps-and-partners-inside-vtex-admin.md

createdAt: "2023-06-04T17:34:00.000Z"
---

>⚠ With the release of Extensions Hub, we changed our terms of service to enable VTEX partners to promote and sell their apps in the App Store. Thus, we created an addendum in the contract with partners, considering the app monetization and the new payment flow.
>

> > Due to changes in our internal procedures, we are with temporary restrictions related to the publishing of apps in the App Store:

Reviewer: ```suggestion
> Due to changes in our internal procedures, we applied temporary restrictions related to publishing apps in the App Store:
```
